contents/blog/raising-3m-for-os.md

```diff
@@ -0,0 +1,230 @@
+
+PostHog chose this route. When we're 80, we think we want to have taken a couple of big swings in our lives even if we don't own the whole pie. On the product side, we can build something open source that will outlast us if we get it wrong, and we think the skills and lessons it will give us will help us in our future careers anyway.
+
+## Bootstrapping
+
+By this we mean getting real customers to pay you money and fuelling your own growth at whatever pace you see fit! Basecamp have a great [philosophy](https://basecamp.com/books/calm) on this. 
+
+You get complete control this way. That might make it easier to do the "right thing", or to be more creative in what you build since you won't have to be disrupting or creating a multi billion dollar market as a constraint. However, the financial pressure of payroll in the early days might make it tough to optimize for long run decisions. You need to feed yourself somehow!
+
+We think there is also a lot of risk in this route too. Humans get tired and hungry. It takes a ton of drive to achieve a profitable company. Giving up would be the major risk. We felt this was a similarly high level of risk to inflating your burn rate above your revenue. That risk is likely worse with open source rather than SaaS - you'll have to manage to build a free product first, then a paid version (unless you go down the hosted/not hosted route, which can create competition from the cloud providers).
+
+Tim and I both quit really good jobs to work on PostHog. We felt we could easily have spent 5 years getting back to a similar salary, which a lot more stress and uncertainty.
+
+## Donations
+
+If it's a side project and you have no need to support yourself, this lets you work at your own pace and see if you can make a little money doing it, unless you're exceptionally popular!
+
+Realistically, it will be very hard to support yourself, let alone to grow a team if you want to build a big business and you have rent to pay. However, this could be a cool way to start before doing bootstrapping or VC in the future.
+
+From this point on, we're going to explain how we raised venture capital in this post.
+
+# How long it took
+
+* August 2019: Tim and I quit our jobs.
+* January 4th 2020: We started the YC batch, and that meant receiving our first $150K investment from YC. We worked on a different idea to start with but soon [pivoted](/blog/pivot-to-posthog).
+* January 23rd 2020: We wrote the first line of code for PostHog
+* February 14th: We did a mini launch for a few YC companies to get early feedback
+* February 21st: PostHog [launched on HackerNews](https://news.ycombinator.com/item?id=22376732)
+* March 6th: Day 1 of fundraising and first cheque ($10K!)
+* March 12th: Left San Francisco due to covid and started working fully remote from the UK. We had around $350K total investment (including the YC money) at this point. Everything seemed to slow down at this point for 3 weeks.
```

Comment:
would be interesting to see more of the steps in between (so it's obvious it slowed down for a while)

Reply:
Todo: add a 'cash counter' for our bank balance!